ui/src/views/Provisioners/ViewWorker/index.jsx

```diff
@@ -288,6 +279,26 @@ export default class ViewWorker extends Component {
                     disabled: actionLoading,
                   }}
                 />
+                {worker.providerId !== NULL_PROVIDER && (
+                  <SpeedDialAction
+                    tooltipOpen
+                    requiresAuth
+                    icon={<DeleteIcon />}
+                    tooltipTitle="Terminate Worker"
```

Comment:
should this tooltip also be dynamic as in WorkerTable?

Reply:
<img width="243" alt="Screen Shot 2022-06-13 at 9 44 00 AM" src="https://user-images.githubusercontent.com/92693437/173367350-6ec0756c-cc49-4c1f-98ab-bcb7b8efc4f3.png">

I decided not to, since the tooltip in this component is essentially the title used describing the button in the speed dial menu. There's no real "tooltip" on hover over these menu options. If that were the case, I'd put the same logic. I felt that just disabling it was sufficient in this case.